app/components/external_links/google_preview_link_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="bs-callout bs-callout-info" id="google-preview" style="display:none;" data="<%= search_item %>">
```

Comment:
It might be a little cleaner/more clear if you specified the data attribute as something like `data-search-item`. Then in `google_preview.js` you could access it via `element.dataset.searchItem` instead of `element.attributes.data.value.toString()`. Not a dealbreaker though; just a style thing.